docs/index.md

```diff
@@ -64,38 +75,47 @@ mkdocs, version 0.14.0
 
 ---
 
-
 ## Getting started
 
 Getting started is super easy.
 
 ```bash
-$ mkdocs new my-project
-$ cd my-project
+mkdocs new my-project
+cd my-project
 ```
 
 Let's take a moment to review the initial project that's been created for us.
 
 ![The initial MkDocs layout](img/initial-layout.png)
 
-There's a single configuration file named `mkdocs.yml`, and a folder named `docs` that will contain our documentation source files.  Right now the `docs` folder just contains a single documentation page, named `index.md`.
+There's a single configuration file named `mkdocs.yml`, and a folder named
+`docs` that will contain our documentation source files. Right now the `docs`
+folder just contains a single documentation page, named `index.md`.
 
-MkDocs comes with a built-in webserver that lets you preview your documentation as you work on it. We start the webserver by making sure we're in the same directory as the `mkdocs.yml` config file, and then running the `mkdocs serve` command:
+MkDocs comes with a built-in webserver that lets you preview your documentation
+as you work on it. We start the webserver by making sure we're in the same
+directory as the `mkdocs.yml` config file, and then running the `mkdocs serve`
+command:
 
 ```bash
 $ mkdocs serve
 Running at: http://127.0.0.1:8000/
```

Comment:
Unlike in other code blocks, I left the `$` in here because output from the command it also shown. See [MD014](https://github.com/mivok/markdownlint/blob/master/docs/RULES.md#md014---dollar-signs-used-before-commands-without-showing-output)